### PR TITLE
Lj 218 update layout with alpha 

### PR DESCRIPTION
**Pull request by MarcoGoC**
# Description

[LJ-218] Migrate layout from Alpha version

Updated our layout using the alpha, fixing some of their issues, 
I may need to update also the hidden navigation, alpha does not have it  

## Test Instructions

1. Download the branch and run 

## Meets Definition of Done

- [ ] Meets design spec
- [ ] unit tests are included

[LJ-218]: https://lj-decd.atlassian.net/browse/LJ-218

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -33,23 +38,26 @@
...
 
 */
 
 
-.header {
+/* .header {
   background-color: #26374a;
   color: white;
-}
+} */
```
Comment (krischarbonneau) [inline styles/globals.css:60]: Can we get rid of this in addition to all the other commented out code in `globals.css`?

File: tailwind.config.js [hunk 2]
```diff
@@ -1,79 +1,130 @@
...
     },
     fontSize: {
-      xxs: ["13px", "16px"],
+      xxs: ["12px", "16px"],
       xs: ["14px", "16px"],
       sm: ["16px", "22px"],
       base: ["18px", "28px"],
       p: ["20px", "30px"],
-      h6: ["19px", "28.5px"],
-      h5: ["20px", "30px"],
```
Comment (krischarbonneau) [inline tailwind.config.js:16]: Getting rid of this results in the `Find information about` text in our topic boxes being too big. Needs to have this sizing as per designs.

**Review by krischarbonneau — APPROVED** [review]
Meaty PR, looks good though I left a couple comments. I assume you didn't bother making the having a child page behave properly since we're restarting that anyway.

The breadcrumbs look like they have a bit too much spacing between them and the `h1` and the report a problem component could use some top margin to better match Canada.ca sites but these are just nitpicky.

File: tailwind.config.js (hunk 2 above)
Comment (MarcoGoC) [reply tailwind.config.js:16]: uh!. those two lines aren't on the alpha site, configuration.  re added

**Review by P0NDER0SA — APPROVED** [review]
This looks good, @MarcoGoC .  tested it and it looks decent as well.

**Pull request merged**